### PR TITLE
[800985596] - Typescript a component of the design system

### PR DESCRIPTION
**Pull request by tomiroth**
Before submitting my PR to Code Review, I have made sure:

- [ ] I have run the tests using `yarn run test`
- [ ] I have exported any new files in index.js
- [ ] My file supports Formik if will potentially be used in a form
- [ ] The PR name follows the convention `(#<ticket_number>) - <ticket_user_story>`
- [ ] I have incremented the version in `package.json` if I want to publish a new version
- [ ] If my change is breaking I have updated the minor version number(i.e. the middle one) and don't merge until I have the PR on Corazon ready

### CODE REVIEW

File: src/components/molecules/FormRadio/FormRadio.tsx [hunk 1]
```diff
@@ -17,9 +20,20 @@ const Container = styled.div`
 `
 Container.displayName = "Container"
 
-const FormRadio = (props) => {
-  const { row, id, className, name, onChange, checked, required, label } = props
 
+export interface IRadioProps{
```
Comment (alexwong-crc) [inline src/components/molecules/FormRadio/FormRadio.tsx:24]: All of these props are now required where as they weren't required before. We will get a lot of issues when we use this on the design system. I feel like name, label and possibly onChanged should be required but the others can be optional. what do you think? 
Or we can stick with the original setup and make them all optional.

File: src/components/molecules/FormRadio/index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
-import FormRadio from "./FormRadio"
```
Comment (alexwong-crc) [inline src/components/molecules/FormRadio/index.js:1]: Good idea, we should probably be exporting the types.

File: src/components/molecules/FormRadio/FormRadio.tsx (hunk 1 above)
Comment (tomiroth) [reply src/components/molecules/FormRadio/FormRadio.tsx:24]: Ah yeah. Good point. I will update that now.

**Review by alexwong-crc — CHANGES_REQUESTED** [review]
You need to bump the design system version if you want to publish the changes.

**Review by alexwong-crc — APPROVED** [review]

**Pull request merged**